pkg/initializer/validator/datavalidator.go

```diff
@@ -68,6 +68,9 @@ func (d *DataValidator) backendPath() string { return filepath.Join(d.snapDir(),
 func (d *DataValidator) Validate(mode Mode, failBelowRevision int64) (DataDirStatus, error) {
 	status, err := d.sanityCheck(failBelowRevision)
 	if status != DataDirectoryValid {
+		if d.OriginalClusterSize > 1 && !miscellaneous.IsBackupBucketEmpty(d.Config.SnapstoreConfig, d.Logger) {
```

Comment:
Why does it matter here if the bucket is empty or not? Why not just return `DataDirStatusUnknownInMultiNode` here if cluster size > 1?

Reply:
Right, makes sense
Thanks for explaining :)